packages/commons/test/BundledAxeScriptProviderTest.cs

```diff
@@ -0,0 +1,45 @@
+using NUnit.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Deque.AxeCore.Commons;
+using Moq;
+using FluentAssertions;
+using System;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Deque.AxeCore.Commons.Test
+{
+    [TestFixture]
+    public class BundledAxeScriptProviderTest
+    {
+        private static readonly string testContext = TestContext.CurrentContext.TestDirectory.ToString();
```

Comment:
`TestContext.CurrentContext` changes as each individual test is running; it would be preferable to avoid storing its data in a static variable whose lifetime outlasts individual test methods'. Here, I think it would be reasonable to just move the statements on L16-17 to be local variables in `getAxeVersion`, since they aren't used anywhere else.